kedro/logging.py

```diff
@@ -1,21 +1,21 @@
-"""
-This module contains a logging handler class which produces coloured logs and tracebacks.
-"""
-
 import logging
 import sys
 from pathlib import Path
 from typing import Any
 
 import click
-import rich.logging
-import rich.pretty
-import rich.traceback
+
+try:
+    import rich.logging
+    import rich.pretty
+    import rich.traceback
+except ImportError:
+    rich: Any = None
```

Comment:
Maybe adding a `RICH` boolean might be cleaner way to do this.

```suggestion
    import rich.traceback
    RICH = True
except ImportError:
    RICH = False
```

Reply:
Can we use importlib instead?

Reply:
As in `importlib.import_module(rich.traceback)` and such?